spec/system/support_interface/docs_spec.rb

```diff
@@ -29,6 +29,8 @@ def then_i_see_the_provider_flow_documentation
 
   def and_it_contains_documentation_for_all_emails
     emails_outside_of_states = %w[
+      candidate_mailer-decline_last_application_choice
+      candidate_mailer-withdraw_last_application_choice
```

Comment:
These can go into the `decline` and `withdraw` events in the YAML.

Reply:
👍